<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateWalletDialog</class>
 <widget class="QDialog" name="CreateWalletDialog">
  <property name="windowTitle">
   <string>Create New Wallet</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="introLabel">
     <property name="text">
      <string>
You are about to create a new wallet.

⚡️ Please write down and secure this 12-word seed phrase. 
It is the ONLY way to recover your wallet if you lose access.
      </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="seedDisplay">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="fontPointSize">
      <number>16</number>
     </property>
     <property name="minimumHeight">
      <number>48</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="copySeedButton">
     <property name="text">
      <string>Copy Seed Phrase</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="confirmLabel">
     <property name="text">
      <string>
Please type the 12 words above in the box below to confirm your backup (separated by spaces):
      </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="seedEntry"/>
   </item>
   <item>
    <widget class="QPushButton" name="confirmButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>I've Saved My Seed Phrase</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <resources/>
 <connections/>
</ui>
